models/skipthoughts.py

```diff
@@ -0,0 +1,194 @@
+"""
+A simple model based on skipthoughts sentence embeddings.
+
+To set up:
+    * Execute the "Getting started" wgets in its README
+    * set config['skipthoughts_datadir'] to directory with downloaded files
+    * make skipthoughts.py from https://github.com/ryankiros/skip-thoughts/blob/master/skipthoughts.py 
+        available via import skipthoughts
```

Comment:
I don't understand what I should do, practically.  Copy that single over? will that work?